mlflow/tracking/fluent.py

```diff
@@ -174,6 +174,23 @@ def active_run():
     return _active_run_stack[-1] if len(_active_run_stack) > 0 else None
 
 
+def get_run(run_id):
+    """
+    Fetch the run from backend store. The resulting :py:class:`Run <mlflow.entities.Run>`
```

Comment:
Nice, would you be able to add similar docstrings to the other methods? e.g. explain what the method does, in addition to describing the params & return type

Reply:
OK. I will do it.
Needs to be done in MLFlowClient in this case as well (where I copied them from)